specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Subscriptions.json

```diff
@@ -100,6 +100,80 @@
                 }
             }
         },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Subscriptions.Admin/moveSubscriptions": {
+            "post": {
+                "x-ms-examples": {
+                    "Move user subscriptions from the Default Provider to a Delegated Provider.": {
+                        "$ref": "./examples/Subscriptions/MoveSubscriptionsToDelegatedProvider.json"
+                    },
+                    "Move user subscriptions from a Delegated Provider to the Default Provider.": {
+                        "$ref": "./examples/Subscriptions/MoveSubscriptionsToDefaultProvider.json"
+                    }
+                },
+                "description": "Move subscriptions between delegated provider offers.",
+                "tags": [
+                    "Subscriptions"
+                ],
+                "operationId": "MoveSubscriptions",
```

Comment:
To be consistent with other operations  make sure there is a prefix for the operations id, in your case `Subscriptions_` which means this operation id become `Subscriptions_MoveSubscriptions` or you can simplify it as `Subscriptions_Move`. Having a prefix will make sure all these operations goes under one class with name `Subscriptions`